src/main/java/com/google/api/gax/grpc/BundlerFactory.java

```diff
@@ -47,6 +47,8 @@
  * a ThresholdBundler, BundleExecutor, and ThresholdBundlingForwarder. The
  * ThresholdBundlingForwarder pulls items from the ThresholdBundler and forwards
  * them to the BundleExecutor for processing.
+ *
+ * <p>This is public only for technical reasons, for advanced usage.
  */
 public class BundlerFactory<RequestT, ResponseT> implements AutoCloseable {
```

Comment:
Sorry I am a bit confused, this class is not marked as final. Are you suggesting that it should be marked as final?